app/uk/gov/hmrc/eusubsidycompliancefrontend/services/UndertakingJourney.scala

```diff
@@ -34,15 +34,14 @@ case class UndertakingJourney(
   name: UndertakingNameFormPage = UndertakingNameFormPage(),
   sector: UndertakingSectorFormPage = UndertakingSectorFormPage(),
   cya: UndertakingCyaFormPage = UndertakingCyaFormPage(),
-  confirmation: UndertakingConfirmationFormPage = UndertakingConfirmationFormPage(),
+  undertakingSuccessDisplay: Boolean = false, // Flag to check if undertaking success screen needs to be displayed
```

Comment:
This is a little bit confusing. What determines when we'd show the success page?

Reply:
Success screen should come up when user selects 'NO' on Add member page during the first time when user has created an undertaking . After that click on No should redirect the user to home account screen like it does now. So undertakingSuccessDisplay flag will be set to true one an undertaking is created , so that user is redirected to success page when they select No on 'Add Member' page. else to the home account screen.